stories/object-status/object-status.stories.js

```diff
@@ -18,62 +18,62 @@ attribute of a line item in a table. `
+<span class="fd-object-status fd-object-status--negative "><i class="fd-object-status__icon sap-icon--status-negative" aria-label="negative Icon" aria-hidden="negative Icon"></i></span>
+<span class="fd-object-status fd-object-status--critical " ><i class="fd-object-status__icon sap-icon--status-critical" aria-label="negative Icon" aria-hidden="negative Icon"></i></span>
+<span class="fd-object-status fd-object-status--positive "><i class="fd-object-status__icon sap-icon--status-positive"  aria-label="Positive Icon" aria-hidden="negative Icon"></i></span>
+<span class="fd-object-status fd-object-status--informative" ><i class="fd-object-status__icon sap-icon--hint" aria-label="Information Icon" aria-hidden="negative Icon"></i></span>
+<span class="fd-object-status " ><i class="fd-object-status__icon sap-icon--to-be-reviewed" aria-label="Review Icon" aria-hidden="negative Icon"></i></span>
 `;
 
 /**
  * Object status text only
  */
 
 export const text = () => `
-<span class="fd-object-status fd-object-status--negative">Negative</span>
-<span class="fd-object-status fd-object-status--critical">Critical</span>
-<span class="fd-object-status fd-object-status--positive">Positive</span>
-<span class="fd-object-status fd-object-status--informative">Informative</span>
-<span class="fd-object-status">Neutral</span>
+<span class="fd-object-status fd-object-status--negative" aria-label="Negative Status " aria-hidden="Negative Status">Negative</span>
+<span class="fd-object-status fd-object-status--critical" aria-label="critical Status " aria-hidden="Critical Status">Critical</span>
+<span class="fd-object-status fd-object-status--positive" aria-label="Positive Status " aria-hidden="Positive Status">Positive</span>
+<span class="fd-object-status fd-object-status--informative" aria-label="Info Status " aria-hidden="Info Status">Informative</span>
+<span class="fd-object-status" aria-label="Neutral Status " aria-hidden="Neutral Status">Neutral</span>
 `;
 
 /**
  * Object status icon and text
  */
 
 export const iconAndText = () => `
-<span class="fd-object-status fd-object-status--negative"><i class="fd-object-status__icon sap-icon--status-negative"></i>Negative</span>
-<span class="fd-object-status fd-object-status--critical"><i class="fd-object-status__icon sap-icon--status-critical"></i>Critical</span>
-<span class="fd-object-status fd-object-status--positive"><i class="fd-object-status__icon sap-icon--status-positive"></i>Positive</span>
-<span class="fd-object-status fd-object-status--informative"><i class="fd-object-status__icon sap-icon--hint"></i>Informative</span>
-<span class="fd-object-status "><i class="fd-object-status__icon sap-icon--to-be-reviewed"></i>Neutral</span>
+<span class="fd-object-status fd-object-status--negative"><i class="fd-object-status__icon sap-icon--status-negative" aria-label="Negative Status with Icon" aria-hidden="Negative Status with Icon"></i>Negative</span>
```

Comment:
`aria-hidden` is a boolean so here it should just be `true`. If `false` you can just leave it out. And since the idea of `aria-hidden="true"` is that the node is purely visual so screen readers can skip it, the `aria-label` should be removed.